cmake/partition_manager.cmake

```diff
@@ -453,8 +453,27 @@ if (CONFIG_SECURE_BOOT AND CONFIG_BOOTLOADER_MCUBOOT)
   # of the symbols used to generate the offset is only available as a
   # generator expression when MCUBoots cmake code exectues. This because
   # partition manager is performed as the last step in the configuration stage.
-  math(EXPR s0_offset "${PM_MCUBOOT_SECONDARY_ADDRESS} - ${PM_S0_ADDRESS}")
-  math(EXPR s1_offset "${PM_MCUBOOT_SECONDARY_ADDRESS} - ${PM_S1_ADDRESS}")
+  if(CONFIG_PM_EXTERNAL_FLASH_MCUBOOT_SECONDARY AND CONFIG_HAS_HW_NRF_QSPI)
+    if(DEFINED ext_flash_dev)
+      get_filename_component(qspi_node ${ext_flash_dev} DIRECTORY)
+    else()
+      dt_nodelabel(qspi_node NODELABEL "qspi")
+    endif()
+	  if(DEFINED qspi_node)
```

Comment:
Wrong alignment, still has tabs

Reply:
What I thought I removed the tabs, my bad seems my editor settings are off....